blueocean-dashboard/src/main/js/components/PipelineRunGraph.jsx

```diff
@@ -73,7 +73,7 @@ function convertJenkinsNodeDetails(jenkinsNode, isCompleted, skewMillis = 0) {
     }
     const i18nDuration = timeManager.format(harmonized.durationInMillis, translate('common.date.duration.hint.format', { defaultValue: 'M [month], d [days], h[h], m[m], s[s]' }));
 
-    const title = translate(`common.state.${state}`, { 0: i18nDuration });
+    const title = state !== 'running' ? translate(`common.state.${state}`, { 0: i18nDuration }) : '';
```

Comment:
So we only want to have the duration as title when it's running? This might be clearer if you reverse the ternary to `state == 'running' ? '' : translate(`common.state.${state}`, { 0: i18nDuration });`

Reply:
No big deal tho, just took me a little bit to parse. But I'm slow :)